en/resources/tutorials/templating.md

```diff
@@ -41,9 +41,14 @@ import PackageDescription
 let package = Package(
     name: "myTemplates",
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.0.0")),
-        .package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", .upToNextMinor(from: "1.0.0")),
-        .package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", .upToNextMinor(from: "1.0.0"))
+        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.2.0")),
+        .package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", .upToNextMajor(from: "1.0.0")),
+        .package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", .upToNextMajor(from: "1.0.0"))
+		]
+		targets: [
+		    .target(name: "myTemplates", dependencies: [ .target(name: "Application"), "Kitura" , "HeliumLogger"]),
+		    .target(name: "Application", dependencies: [ "Kitura", "KituraStencil", "KituraMustache" ])
+		]
  ])
```

Comment:
Extra closing square bracket that needs to be removed

Reply:
Another good spot, now fixed.